projects/packages/assets/src/class-assets.php

```diff
@@ -343,6 +367,7 @@ public static function register_script( $handle, $path, $relative_to, array $opt
 			'media'            => 'all',
 			'minify'           => false,
 			'textdomain'       => null,
+			'strategy'         => '',
```

Comment:
Alphabetical order again.
```suggestion
			'strategy'         => '',
			'textdomain'       => null,
```